mods/lord/World/sunshine/src/sunshine.lua

```diff
@@ -0,0 +1,27 @@
+require('commands')
+
+local lighting_default = {
```

Comment:
```suggestion
local default_lighting = {
```
так оно более по-английски.
однако это тоже можно в тот самый api закинуть и там будет `lighting.default = {...}`